stream-portfolio/portfolio-bootstrap-task/src/main/resources/application.yml

```diff
@@ -0,0 +1,25 @@
+# Stream config
+spring:
+  zipkin:
+    enabled: false
+  security:
+    oauth2:
+      client:
+        registration:
+          dbs:
```

Comment:
The new client name is now `bb`, the old one is still retro compatible but we should try to keep using the new one from now on